docs/getting_started/requirements.md

```diff
@@ -10,26 +10,28 @@ The following server requirements cover both running the software on-premise and
 
     For running on [Ibexa Cloud](https://www.ibexa.co/products/ibexa-cloud), where recommended configuration and support is provided out of the box, see separate [Ibexa Cloud section](#ibexa-cloud-requirements-and-setup) for further reading on its requirements.
 
-The minimal setup requires PHP,  MySQL/MariaDB, Apache/Nginx, Node.js and `yarn`.
+The minimal setup requires PHP, MySQL/MariaDB, Apache/Nginx, Node.js and `yarn`.
 Recommendation for production setups is to use Varnish/Fastly, Redis/Memcached, NFS/EFS/S3 and Solr/Elasticsearch in a [clustered setup](clustering.md).
 
-Using the latest listed version of each product or component is always recommended.
+!!! note "Recommended versions"
+
+    Using the latest listed version of each product or component is always recommended. Review all the recommended versions carefully. If you see a "+" next to the product version it means that we recommend this version or higher within the same major release for example "Nginx 1.18+" means any 1.x version higher or equal to 1.18, but not 2.x.
 
 ## Operating system
 
 - Debian 10.x "buster" or Debian 11.x "bullseye"
-- Ubuntu 20.04 "Focal Fossa"
+- Ubuntu 20.04 "Focal Fossa" or higher
```

Comment:
In case of operating systems "or higher" is a loaded statement. Because an OS includes in itself many versions of various components.
In the particular case of Ubuntu, 20.04 "Focal Fossa" is the previous LTS (still in standard support, as is 18.04), 22.04 "Jammy Jellyfish" is the current LTS. But it hasn't been tested with our product yet.
In short, IMO we shouldn't add this until the lates LTS was tested (in a separate sprint ticket).
ref. https://www.releases.ubuntu.com/